test/adapter/json/polymorphic_test.rb

```diff
@@ -0,0 +1,25 @@
+require 'test_helper'
+
+module ActiveModel
+  class Serializer
+    module Adapter
+      class Json
+        class PolymorphicTest < ActiveSupport::TestCase
+          def setup
```

Comment:
@beauby nothing wrong, but we're taking advantage of activesupport::testcase which has this feature (just replace `s/def setup/setup do`)